actix-http/src/cookie/parse.rs

```diff
@@ -179,13 +179,13 @@ fn parse_inner<'c>(s: &str, decode: bool) -> Result<Cookie<'c>, ParseError> {
                 }
             }
             ("expires", Some(v)) => {
-                // Try strptime with three date formats according to
+                // Try parsing with three date formats according to
                 // http://tools.ietf.org/html/rfc2616#section-3.3.1. Try
                 // additional ones as encountered in the real world.
-                let tm = time::strptime(v, "%a, %d %b %Y %H:%M:%S %Z")
-                    .or_else(|_| time::strptime(v, "%A, %d-%b-%y %H:%M:%S %Z"))
-                    .or_else(|_| time::strptime(v, "%a, %d-%b-%Y %H:%M:%S %Z"))
-                    .or_else(|_| time::strptime(v, "%a %b %d %H:%M:%S %Y"));
+                let tm = PrimitiveDateTime::parse(v, "%a, %d %b %Y %H:%M:%S")
```

Comment:
this needs to be tested, I think this will fail to parse date with timezone

Reply:
It doesn't, it simply ignores the timezone, which is identical to the behavior observed even while using %Z. For example, this code:

```Rust
let dt_a = PrimitiveDateTime::parse("Sun, 07 Nov 1994 08:48:37 PST", "%a, %d %b %Y %H:%M:%S").unwrap();
let dt_b = PrimitiveDateTime::parse("Sun, 07 Nov 1994 08:48:37 UTC", "%a, %d %b %Y %H:%M:%S").unwrap();

println!("{:?}", dt_a);
println!("{:?}", dt_b);
```

outputs the following:

```Rust
PrimitiveDateTime { date: Date { year: 1994, ordinal: 311 }, time: Time { hour: 8, minute: 48, second: 37, nanosecond: 0 } }
PrimitiveDateTime { date: Date { year: 1994, ordinal: 311 }, time: Time { hour: 8, minute: 48, second: 37, nanosecond: 0 } }
```